<ui version="4.0" >
 <class>QtMISolverDialogBase</class>
 <widget class="QDialog" name="QtMISolverDialogBase" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="1" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Max Error (1e-x)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="3" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Iterations</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Local</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QDoubleSpinBox" name="localErrorDSB" >
       <property name="decimals" >
        <number>2</number>
       </property>
       <property name="minimum" >
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum" >
        <double>1000000000.000000000000000</double>
       </property>
       <property name="value" >
        <double>6.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="3" >
      <widget class="QSpinBox" name="localItersSB" >
       <property name="maximum" >
        <number>1000000000</number>
       </property>
       <property name="value" >
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>CG</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QDoubleSpinBox" name="cgErrorDSB" >
       <property name="decimals" >
        <number>2</number>
       </property>
       <property name="minimum" >
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum" >
        <double>1000000000.000000000000000</double>
       </property>
       <property name="value" >
        <double>6.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="3" >
      <widget class="QSpinBox" name="cgItersSB" >
       <property name="maximum" >
        <number>1000000000</number>
       </property>
       <property name="value" >
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Full</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QDoubleSpinBox" name="fullErrorDSB" >
       <property name="decimals" >
        <number>2</number>
       </property>
       <property name="minimum" >
        <double>-1000000000.000000000000000</double>
       </property>
       <property name="maximum" >
        <double>1000000000.000000000000000</double>
       </property>
       <property name="value" >
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="3" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QCheckBox" name="finalFullCB" >
         <property name="text" >
          <string>final</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <property name="tristate" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="initialFullCB" >
         <property name="text" >
          <string>initial</string>
         </property>
         <property name="checked" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Info Level</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QSpinBox" name="infoSB" />
     </item>
     <item row="4" column="2" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Rounding</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3" >
      <widget class="QCheckBox" name="directRoundingCB" >
       <property name="text" >
        <string>direct</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4" >
      <widget class="QCheckBox" name="noRoundingCB" >
       <property name="text" >
        <string>XnoX</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QPushButton" name="okPB" >
       <property name="text" >
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="solverStatsCheckBox" >
     <property name="text" >
      <string>Output solver statistics</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
